compiler/rustc_attr_parsing/src/attributes/doc.rs

```diff
@@ -0,0 +1,571 @@
+                                        sub_item.path().span(),
+                                        a.name_value(),
+                                        sub_item.span(),
+                                        cx,
+                                    )
+                                {
+                                    cfg_hide_show.values.push(CfgInfo { name, name_span, value })
+                                }
+                            }
+                            _ => {
+                                cx.emit_lint(
+                                    AttributeLintKind::DocAutoCfgHideShowUnexpectedItem {
+                                        attr_name,
+                                    },
+                                    sub_item.span(),
+                                );
+                                continue;
+                            }
+                        }
+                    }
+                    self.attribute.auto_cfg.push((cfg_hide_show, path.span()));
+                }
+            }
+            ArgParser::NameValue(nv) => {
+                let MetaItemLit { kind: LitKind::Bool(bool_value), span, .. } = nv.value_as_lit()
+                else {
+                    cx.emit_lint(AttributeLintKind::DocAutoCfgWrongLiteral, nv.value_span);
+                    return;
+                };
+                self.attribute.auto_cfg_change.push((*bool_value, *span));
+            }
+        }
+    }
+
+    fn parse_single_doc_attr_item<'c, S: Stage>(
+        &mut self,
+        cx: &'c mut AcceptContext<'_, '_, S>,
+        mip: &MetaItemParser<'_>,
+    ) {
+        let path = mip.path();
+        let args = mip.args();
+
+        macro_rules! no_args {
+            ($ident: ident) => {{
+                if let Err(span) = args.no_args() {
+                    cx.expected_no_args(span);
+                    return;
+                }
+
+                if self.attribute.$ident.is_some() {
+                    cx.duplicate_key(path.span(), path.word_sym().unwrap());
+                    return;
+                }
+
+                self.attribute.$ident = Some(path.span());
+            }};
+        }
+        macro_rules! string_arg {
+            ($ident: ident) => {{
+                let Some(nv) = args.name_value() else {
+                    cx.expected_name_value(args.span().unwrap_or(path.span()), path.word_sym());
+                    return;
+                };
+
+                let Some(s) = nv.value_as_str() else {
+                    cx.expected_string_literal(nv.value_span, Some(nv.value_as_lit()));
+                    return;
+                };
+
+                // FIXME: It's errorring when the attribute is passed multiple times on the command
+                // line.
+                // if self.attribute.$ident.is_some() {
+                //     cx.duplicate_key(path.span(), path.word_sym().unwrap());
+                //     return;
+                // }
```

Comment:
Does leaving this out cause a regression? If so please fix, otherwise we can leave it like this

Reply:
No, the opposite. If we uncomment it, `bootstrap` command will fail because rustdoc args are duplicated. Gonna need to fix that.